ReText/window.py

```diff
@@ -823,6 +826,17 @@ def openFile(self):
 	def openFileWrapper(self, fileName):
 		if not fileName:
 			return
+		if not QFile.exists(fileName):
+			if self.createFileQuestion(fileName):
+				try:
+					print('Create file' + ' ' + fileName , file=sys.stdout)
```

Comment:
If this is debug print, delete it. If you think it's useful then please:

- Remove `file=sys.stdout`, it is default anyway.
- Use a format function, like `'Create file %s' % fileName`, for readability.